content/docs/quickstart/kubernetes/_index.md

```diff
@@ -6,12 +6,32 @@ menu:
     identifier: kubernetes
     name: Kubernetes
     weight: 2
+cloudlink: /docs/reference/clouds/kubernetes
 ---
+Learn how to quickly provision infrastructure on Kubernetes with Pulumi using JavaScript, TypeScript, or Python as your language runtime.
```

Comment:
I'm not sure this paragraph is adding much value by itself; it feels like its message (use languages) could be worked into the paragraph below it somehow.

Reply:
This is the intro sentence I'm hoping to use for all the other Get Started guides. I think it will be nice to know what's coming up (i.e. what example languages are available in the guides, but we could express that via language icons instead).

Reply:
Yep, I think so too -- the visual indicators we're considering should be able to get this point across.